### PR TITLE
Add mypy typings

### PR DESCRIPTION
**Pull request by ObjatieGroba**
Python projects that uses debinterface would be able check it with mypy

### CODE REVIEW

**Comment by ObjatieGroba** [comment]
Would it better to do it on 3.5.0 version?

**Review by nMustaki — CHANGES_REQUESTED** [review]
I'm all to put into 3.5, it's a great feature, the sooner the better !

File: debinterface/interfaces.py [hunk 1]
```diff
@@ -5,15 +5,18 @@
 from .interfacesReader import InterfacesReader
 from .adapter import NetworkAdapter
 from . import toolutils
+import typing as tp
```
[the remark this answers is not in the document]
Comment (ObjatieGroba) [reply debinterface/interfaces.py:8]: Should it be?

Python 3.5 and later contains typing as build-in module. For other it is possible to specify `typing;python_version<"3.5"` requirement.

I do not really sure what is the best way

[the remark this answers is not in the document]
Comment (nMustaki) [reply debinterface/interfaces.py]: I'd like to avoid adding a dependency if possible, as long as we support python2

**Review by nMustaki — APPROVED** [review]

**Pull request merged**